src/3rdparty/kissfft/Makefile

```
@@ -1,48 +1,323 @@
+
+%.c.o: %.c
+	$(CC) -Wall -fPIC \
+		-o $@ \
+		$(CFLAGS) $(TYPEFLAGS) -DKISS_FFT_BUILD \
+		-c $<
+
+#
+# Target: "make all"
+#
+
+all: kfc.c.o kiss_fft.c.o kiss_fftnd.c.o kiss_fftndr.c.o kiss_fftr.c.o
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)